codecov_auth/commands/owner/interactors/save_terms_agreement.py

```diff
@@ -30,6 +32,13 @@ def update_terms_agreement(self, input: TermsAgreementInput):
         self.current_user.terms_agreement_at = timezone.now()
         self.current_user.save()
 
+        OwnerProfile.objects.update_or_create(
```

Comment:
After our conversation in Slack, we wouldn't need this and instead would be `self.current_user.customer_intent = input.customer_intent`